theme/less/_navigation.less

```diff
@@ -93,13 +95,16 @@
 #main-navigation li.has-children > a:after
 {
   content: url('/images/arrow_bullet.png');
-  padding-left: 10px;
+  padding-left: 11px;
+  vertical-align: sub;
```

Comment:
This aligns `a:after` as subscript of the text. Instead of this could be for example  `vertical-align` with negative `em` or percentage values to align image properly.

Reply:
This is how it looks like, on my side.

<img width="1002" alt="Screen Shot 2020-08-04 at 10 15 29 PM" src="https://user-images.githubusercontent.com/716969/89334962-0548f400-d6a0-11ea-80ba-9efa16b92e5e.png">

switching this to `vertical-align: middle;` seem to fix the arrow to be aligned in the middle of the text.

Reply:
Also seems like items without side arrow are missing some paddings.

Reply:
LGTM.